homeassistant/util/package.py

```diff
@@ -14,6 +14,7 @@
 from urllib.parse import urlparse
 
 from packaging.requirements import InvalidRequirement, Requirement
+from pip._internal.models.link import Link
```

Comment:
I'm not sure if we should use this. Internal pip APIs are not part of pip's public API and should generally not be consumed.